cmd/libsnap-confine-private/utils.c

```diff
@@ -28,6 +28,15 @@
 #include "panic.h"
 #include "utils.h"
 
+int sc_nonblocking_stdout(void)
+{
+    int flags = fcntl(STDOUT_FILENO, F_GETFL, 0);
```

Comment:
Silly question - isn't our debug output going to stderr?

Reply:
That's interesting, absolutely correct. I don't know how I didn't notice I was testing with 2>&1. Let me just correct this. Thank you for catching this.